api/policies/isInGame.js

```diff
@@ -16,5 +16,5 @@ module.exports = function (req, res, next) {
     }
     return res.badRequest({ message: "Error with user's session: 'game' must be an integer" });
   }
-  return res.badRequest({ message: 'Session Error: You must join a game to perform that move' });
+  return res.badRequest({ message: "You can't do that because you're not a player in this game" });
```

Comment:
```suggestion
  return res.badRequest({ message: 'You can not do that because you are not a player in this game' });
```
I think removing the contractions is also an ok middle ground, doesn't change much unless you want it more informal. Could also use backticks.

Reply:
Yeah that's reasonable. I kinda like the informality of the contractions but mixed quote types are funky

Reply:
Personally I like the contractions but wanted to call out consistency around the strings. Bsckticks would be a good middle ground.